package.json

```diff
@@ -64,96 +68,99 @@
+    "test-fast": "mocha --file ./build/compiled/test/utils/test-setup.js --bail --recursive --timeout 90000 ./build/compiled/test",
+    "watch": "./node_modules/.bin/tsc -w"
+  },
+  "dependencies": {
+    "@sqltools/formatter": "^1.2.5",
+    "ansis": "^3.17.0",
+    "app-root-path": "^3.1.0",
+    "buffer": "^6.0.3",
+    "dayjs": "^1.11.13",
+    "debug": "^4.4.0",
+    "dotenv": "^16.4.7",
+    "glob": "^10.4.5",
+    "sha.js": "^2.4.11",
+    "sql-highlight": "^6.0.0",
+    "tslib": "^2.8.1",
+    "uuid": "^11.1.0",
+    "yargs": "^17.7.2"
   },
-  "homepage": "https://typeorm.io",
-  "tags": [
-    "orm",
-    "typescript",
-    "typescript-orm",
-    "mysql",
-    "mysql-orm",
-    "postgresql",
-    "postgresql-orm",
-    "mariadb",
-    "mariadb-orm",
-    "spanner",
-    "sqlite",
-    "sqlite-orm",
-    "sql-server",
-    "sql-server-orm",
-    "oracle",
-    "oracle-orm",
-    "cloud-spanner",
-    "cloud-spanner-orm"
-  ],
   "devDependencies": {
-    "@eslint/js": "^9.17.0",
-    "@tsconfig/node16": "^16.1.1",
-    "@types/chai": "^4.3.4",
-    "@types/chai-as-promised": "^7.1.5",
-    "@types/debug": "^4.1.7",
+    "@eslint/js": "^9.22.0",
+    "@tsconfig/node16": "^16.1.3",
+    "@types/chai": "^4.3.20",
+    "@types/chai-as-promised": "^7.1.8",
+    "@types/debug": "^4.1.12",
     "@types/gulp-rename": "^2.0.6",
     "@types/gulp-sourcemaps": "^0.0.38",
-    "@types/mocha": "^10.0.1",
-    "@types/node": "^18.13.0",
-    "@types/sha.js": "^2.4.0",
-    "@types/sinon": "^10.0.13",
-    "@types/source-map-support": "^0.5.6",
-    "@types/uuid": "^9.0.0",
-    "@types/yargs": "^17.0.22",
-    "better-sqlite3": "^8.1.0",
-    "chai": "^4.3.7",
-    "chai-as-promised": "^7.1.1",
+    "@types/mocha": "^10.0.10",
+    "@types/node": "^16.18.126",
+    "@types/sha.js": "^2.4.4",
+    "@types/sinon": "^10.0.20",
+    "@types/source-map-support": "^0.5.10",
+    "@types/yargs": "^17.0.33",
+    "better-sqlite3": "^8.7.0",
+    "chai": "^4.5.0",
+    "chai-as-promised": "^7.1.2",
     "class-transformer": "^0.5.1",
-    "conventional-changelog-angular": "^5.0.13",
-    "conventional-changelog-cli": "^2.2.2",
-    "eslint": "^9.17.0",
-    "globals": "^15.14.0",
+    "conventional-changelog-angular": "^7.0.0",
+    "conventional-changelog-cli": "^5.0.0",
+    "eslint": "^9.22.0",
+    "globals": "^16.0.0",
     "gulp": "^4.0.2",
     "gulp-istanbul": "^1.1.3",
-    "gulp-mocha": "^10.0.0",
+    "gulp-mocha": "^10.0.1",
     "gulp-rename": "^2.0.0",
     "gulp-replace": "^1.1.4",
     "gulp-shell": "^0.8.0",
     "gulp-sourcemaps": "^3.0.0",
     "gulp-typescript": "^6.0.0-alpha.1",
     "gulpclass": "^0.2.0",
-    "husky": "^8.0.3",
     "mocha": "^10.8.2",
-    "mongodb": "^6.3.0",
+    "mongodb": "^6.14.2",
     "mssql": "^11.0.1",
     "mysql": "^2.18.1",
-    "mysql2": "^3.9.7",
+    "mysql2": "^3.13.0",
     "nyc": "^17.1.0",
```

Comment:
`nyc` has minimum requirement `node@18`. @OSA413 not sure if this was affecting the Oracle tests.

Keeping the latest version though, maybe we'll also drop support for `node@16` soon, it reached EOL anyway, and so will `node@18` very soon.